erpnext/controllers/accounts_controller.py

```diff
@@ -541,3 +524,20 @@ def _on_previous_row_error(row_range):
 			_on_previous_row_error("1 - %d" % (tax.row_id,))
 		elif tax.get("category") == "Valuation":
 			frappe.throw(_("Valuation type charges can not marked as Inclusive"))
+
+def set_balance_in_account_currency(gl_dict, conversion_rate=None, company_currency=None, account_currency=None):
```

Comment:
Don't change the position of the argument "account_currency", keep it in the second place.